modules/ROOT/pages/install-aws.adoc

```diff
@@ -236,12 +243,16 @@ docker run -it -v $(pwd):/src -w /src/aws \
   -var mule_license='' \
   -state=tf-data/rtf.tfstate
 ----
-
++
 The Terraform script provisions the infrastructure and runs the installation script on each VM. When complete, 
 the Runtime Fabric updates to Active status in Runtime Manager.
-
++
 [NOTE]
-This step installs Runtime Fabric across all servers to form a cluster. It may take 15-25 minutes or longer to complete.
+====
+
+* This step installs Runtime Fabric across all servers to form a cluster. It may take 15-25 minutes or longer to complete.
```

Comment:
If this is truly a list item
```suggestion
* This step installs Runtime Fabric across all servers to form a cluster. 
It might take 15 to 25 minutes or longer to finish.
```